modules/monitoring-specifying-how-a-service-is-monitored-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="specifying-how-a-service-is-monitored-for-cluster-observability-operator_{context}"]
+= Specifying how a service is monitored for Cluster Observability Operator
+
+To use the metrics exposed by your sample service, you must configure monitoring components to scrape metrics from the `/metrics` endpoint. 
+
+You can create this configuration by using a `ServiceMonitor` custom resource definition (CRD) that specifies how the service should be monitored, or a `PodMonitor` CRD that specifies how a pod should be monitored. 
```

Comment:
I would check this with your eng team, but I would think these are custom resources (CRs) or objects rather than CRDs. Could be wrong, though.

Reply:
@lihongyan1 @jan--f - Are they CRDs or CRs?

Reply:
Both `ServiceMonitor` and `PodMonitor` are CRDs, here you create CRs based these CRD